### PR TITLE
a few initial changes...

### PR DESCRIPTION
**Pull request by jorgepz**

### CODE REVIEW

**Comment by jorgepz** [comment]
I propose using the existing gauss points function. This will probably require some changes in the plasticity examples to make it work...

**Comment by JoaquinViera** [comment]
Ok, i will check the existing gauss function to adapt the code

**Review by JoaquinViera — APPROVED** [review]

File: examples/staticVonMisesTruss/onsasExample_staticVonMisesTruss.m [hunk 1]
```diff
@@ -107,7 +107,7 @@
 %md
 %md### Analysis case 1: Newton-Raphson with Rotated Eng Strain
 %md In the first case ONSAS is run and the solution at the dof of interest is stored.
-[matUs, loadFactorsMat] = ONSAS( materials, elements, boundaryConds, initialConds, mesh, analysisSettings, otherParams ) ;
+[matUs, loadFactorsMat, ~] = ONSAS( materials, elements, boundaryConds, initialConds, mesh, analysisSettings, otherParams ) ;
```
Comment (JoaquinViera) [inline examples/staticVonMisesTruss/onsasExample_staticVonMisesTruss.m:110]:
```suggestion
[matUs, loadFactorsMat ] = ONSAS( materials, elements, boundaryConds, initialConds, mesh, analysisSettings, otherParams ) ;
```

Comment (jorgepz) [reply examples/staticVonMisesTruss/onsasExample_staticVonMisesTruss.m:110]: ok

**Pull request merged**